policy/pipeline/required_tasks.rego

```diff
@@ -0,0 +1,34 @@
+package policy.pipeline.required_tasks
+
+import data.lib
+
+# METADATA
+# title: Check the pipline contains required check tasks
+# description: |-
+#   Every build pipeline is expected to contain a set of checks and tests that
+#   are required by Enterprise Contract. This rule confirms that the pipeline
+#   definition includes all the expected tasks.
+#
+#   The matching is done using the taskRef name rather than the task name.
```

Comment:
Could say "rather than the pipeline task name."

Reply:
Done.